RELEASE_NOTES.md

```diff
@@ -34,45 +34,46 @@ System Compatibility
-- Added Python version for fpgadiag
-- Implemented Python distributions to be generated for pyfpgadiag, pypackager and pyfpgaflash
-- Defined and implemented SysObject API
-- Increased test coverage
+- Updated python requests package used by Sphinx
+- Updated fpgad to enumerate for supported devices discarding the previous assumption that it is running on a dual-socket integrated FPGA platform
+- Added Python version of fpgamux
+- Added deprecation notice for legacy C++ API
+- Updated default installation location for packages generated through CMake to "/usr" instead of "/usr/local"
+- Added support to FPGA Linux kernel Device Feature List (DFL) driver patch set1. (see Notes below for more information)
+- Updated RPM package
+- Increased test cases and test coverage
 - Cleaned up dead/legacy code
 - Various bug fixes
+- Various warning fixes
 - Various memory leak fixes
 
 Notes / Known Issues
 --------------------
--   OPAE is transitioning to a plugin-centric model. What was the opae-c library in
-prior releases now exists as a local plugin, named xfpga. The new opae-c library
-is an API shell that discovers and utilizes plugins. This enables supporting multiple platforms simultaneously through dynamic discovery and loading of plugins. For more information, refer
-to the [plugin developer's guide](https://opae.github.io/latest/docs/fpga_api/plug_guide/readme.html). 
+-  libopaec++ is being deprecated in favor of the official OPAE C++ API.
+While no tools in the OPAE codebase use libopaec++, the code is being kept here
+for any other tools that may use it outside of the OPAE repository.
+This directory, however, will be removed in future versions of OPAE.
+For more information and reference on the official API, see the
+[documentation](https://opae.github.io/latest/docs/fpga_api/fpga_cxx_api.html).
 
--   Metrics API / fpgainfo tool require latest FIM and drivers for PAC platforms
+-   fpgamux has been ported to use the supported OPAE C++ API (in addition to Python version).
 
--   fpgadiag has also been ported to use the supported OPAE C++ API (in addition to Python version).
-    fpgamux has not been ported but will be ported in a future release.
-    The C++ library previously used will be deprecated.
-
--   Any SysObject names used with the SysObject API are driver specific and may not be compatible accross plugins and/or drivers.
+-   In addition to supporting the OPAE driver bundled with OPAE SDK releases, the OPAE SDK libraries now
+support the FPGA driver that has been upstreamed to the Linux Kernel 4.18.
+For more details on this, please see the [OPAE documentation](https://opae.github.io/1.3.0/docs/fpga_dfl_drv/fpga_dfl_drv.html) related to this.
 
 -   Seldom in stress tests, kernel panic may be encountered in kernel version 3.10. Preliminary debug information seems to indicate it may be related to hugepage support in the Linux kernel.
 
+-   The current Python distributions included in this release are
```

Comment:
This python package update should not be in the known issue part, right?

Reply:
It seems to me that it should be in the changes part.

Reply:
I think it fits here because it is not something that changed from the last release. It is just a way to let users know that those are the python distributions with this release.